src/components/DataInput/DropDownSelect.tsx

```diff
@@ -40,4 +42,26 @@ const DropdownListInput = <Value,>(props: Props<Value>): React.ReactElement => {
     );
 };
 
-export default DropdownListInput;
+export const ControlledSelect = <Value,>(props: Props<Value>): React.ReactElement => {
+    return (
+        <GenericSelect
+            selectLabelId={props.selectLabelId}
+            listTitle={props.listTitle}
+            value={props.value}
+            labelsAndValues={props.labelsAndValues}
+            onChange={props.onChange}
+        />
+    );
+};
+
+export const UncontrolledSelect = <Value,>(props: Props<Value>): React.ReactElement => {
+    return (
+        <GenericSelect
+            selectLabelId={props.selectLabelId}
+            listTitle={props.listTitle}
+            defaultValue={props.defaultValue}
+            labelsAndValues={props.labelsAndValues}
+            onChange={props.onChange}
+        />
+    );
+};
```

Comment:
Can we rename `Props` to `GenericProps` then have two different sets of props for the two components based on what they can and can't accept?

While we're at it, let's rename `Value` to `ValueType` for clarity